src/unix/posix.h

```diff
@@ -80,4 +80,14 @@ GIT_INLINE(int) p_futimes(int f, const struct p_timeval t[2])
 # define p_futimes futimes
 #endif
 
+#ifdef HAVE_REGCOMP_L
+#include <xlocale.h>
+GIT_INLINE(int) p_regcomp(regex_t *preg, const char *pattern, int cflags)
+{
+	return regcomp_l(preg, pattern, cflags, (locale_t) 0);
```

Comment:
If `(locale_t) 0` defined somewhere to be special? I've found `LC_GLOBAL_LOCALE` in the headers which is defined as `(locale_t) -1` but not for `0`.